examples/tracing/otel-collector-config.yml

```diff
@@ -0,0 +1,32 @@
+receivers:
+  zipkin:
+    endpoint: 0.0.0.0:9411
```

Comment:
Suggest adding OTLP receiver as well even if example app doesn't yet export in OTLP